browser/resources/settings/brave_tor_page/brave_tor_browser_proxy.ts

```diff
@@ -3,65 +3,64 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at https://mozilla.org/MPL/2.0/.
 
-// @ts-nocheck TODO(petemill): Define types and remove ts-nocheck
-
 import { sendWithPromise } from 'chrome://resources/js/cr.js';
 
+// TODO(petemill): Define the expected types instead of using any
 export interface BraveTorBrowserProxy {
-  getBridgesConfig(): Promise<any> // TODO(petemill): Define the expected type
-  setBridgesConfig(config: any) // TODO(petemill): Define the expected type
-  requestBridgesCaptcha(): Promise<any> // TODO(petemill): Define the expected type
-  resolveBridgesCaptcha(captcha: any) // TODO(petemill): Define the expected type
-  setTorEnabled(value: boolean)
-  isTorEnabled(): Promise<boolean>
-  isTorManaged(): Promise<boolean>
-  isSnowflakeExtensionAllowed(): Promise<boolean>
-  isSnowflakeExtensionEnabled(): Promise<boolean>
-  enableSnowflakeExtension(enable: boolean): Promise<boolean>
+  getBridgesConfig: () => Promise<any>
+  setBridgesConfig: (config: any) => void
+  requestBridgesCaptcha: () => Promise<any>
+  resolveBridgesCaptcha: (captcha: any) => Promise<any>
+  setTorEnabled: (value: boolean) => void
+  isTorEnabled: () => Promise<boolean>
+  isTorManaged: () => Promise<boolean>
+  isSnowflakeExtensionAllowed: () => Promise<boolean>
+  isSnowflakeExtensionEnabled: () => Promise<boolean>
+  enableSnowflakeExtension: (enable: boolean) => Promise<boolean>
 }
 
 export class BraveTorBrowserProxyImpl implements BraveTorBrowserProxy {
   static getInstance() {
     return instance || (instance = new BraveTorBrowserProxyImpl());
   }
 
-  getBridgesConfig() {
+  getBridgesConfig(): Promise<any> {
```

Comment:
`sendWithPromise` already returns a `Promise<any>`
```suggestion
  getBridgesConfig() {
```

Reply:
Fixed here https://github.com/brave/brave-core/pull/27107/commits/6e7d181bb544acdef5b2846411676bff041b8c7a, thanks!